USAGE.md

```diff
@@ -0,0 +1,386 @@
+
+**Enabling the Kill Switch**
+
+To enable the Kill Switch, open the configuration menu with `protonvpn configure`, then select `5` for the Kill Switch and confirm the activation with `y`. On the next connection the Kill Switch will be enabled.
+
+*Note: The Kill Switch only activates on unexptected connection drops. It will not persist through reboots and not activate when calling `protonvpn disconnect`. To simulate the Kill Switch, kill the OpenVPN process while connected with `sudo pkill openvpn`.*
+
+### Split Tunneling
+
+ProtonVPN-CLI features IP-based split tunneling. This means that you can exclude specific IPs or networks from being routed over the VPN tunnel.
+
+*Note: Split Tunneling does not work when the Kill Switch is enabled.*
+
+**Enable Split Tunneling**
+
+To enable Split Tunneling, open the configuration menu with `protonvpn configure`, then select Split Tunneling with `6`. Then confirm with `y`.
+
+Now add the IPs you want (one IP at a time) or networks in [CIDR notation](https://www.ipaddressguide.com/cidr).
+
+If you want to have a big list of IPs or networks that you want to exclude, it is recommended to add one IP via above mentioned method, this will create the file `~/.pvpn-cli/split_tunnel.txt`. You can then paste the IPs or networks in CIDR notation in this file, one IP/network per line.
+
+Then call `protonvpn refresh` to update the OpenVPN template with your excluded IP addresses.
+
+## Enhancements
+
+This is a list of optional enhancements that make using ProtonVPN-CLI easier.
+
+### Disable sudo password query
+
+You can disable the prompt for the sudo password when using ProtonVPN-CLI.
+
+1. Enter `sudo which protonvpn` to find where the executable is installed
+
+    ![which-protonvpn](https://i.imgur.com/JjYpviI.png)
+
+2. Enter `sudo visudo` to edit the sudoers file.
+
+    Go to the bottom of the file and add the following line
+
+    `user ALL = (root)  NOPASSWD: /usr/local/bin/protonvpn`
+
+    *Note: Make sure to replace the name `user` with your own username and the path with the output of the previous command*
+
+3. Save the file
+
+Now you can use ProtonVPN-CLI without entering your password. This is best used with an [alias](#configure-alias-for-quicker-access).
+
+### Configure alias for quicker access
+
+An alias let's you access the `protonvpn` command easier and without typing `sudo` all the time. To configure an alias, open your shell's rc file with your favorite editor, for bash, which is the default shell on most linux distributions, this would be `~/.bashrc`. Now add the following lines at the end of the file
+
+`alias protonvpn='sudo protonvpn'`
+
+`alias pvpn='sudo protonvpn'`
+
+This let's you use ProtonVPN-CLI by just typing `protonvpn` without sudo or even just typing `pvpn`. For the latter, make sure you have uninstalled the old [bash-based ProtonVPN-CLI](https://github.com/ProtonVPN/protonvpn-cli) to avoid complications.
```

Comment:
Same here, "lets" not "let's".
```suggestion
This lets you use ProtonVPN-CLI by simply typing `protonvpn` without sudo or just typing `pvpn`. For the latter, make sure you have uninstalled the old [bash-based ProtonVPN-CLI](https://github.com/ProtonVPN/protonvpn-cli) to avoid complications.
```